server/routes/products.js

```diff
@@ -2,5 +2,7 @@ const router = require('express').Router();
 const controller = require('../controllers/products');
 
 router.get('/', controller.get);
```

Comment:
move '/' line to the bottom. I think David was saying that the more generic endpoint should be at the bottom and the more specific endpoints should be at the top

Reply:
Changed it now